guidelines/releases-guidelines/02-release-process.md

```diff
@@ -3,39 +3,65 @@ title: Release process
 label: internal
 ---
 
-This document describes how to create a Kyma release using Prow.
+This document explains how a new Kyma and also new Kyma CLI release can be introduced.
 
-## Preparation
+## Kyma
 
-This section only applies to new major and minor versions. If you release a patch, skip the preparation and go to the [**Steps**][/#release-process-release-process-steps] section.
+### Release Information
```

Comment:
Why are we removing this? I think the statement is still valid but the link is broken.

Reply:
There are dedicated sections for `#### Steps for new major/minor version` and `#### Steps for new release candidate`. Therefore I thought we have it twice if we keep the statement